src/plugins/telemetryTable/collections/SortedTableRowCollection.js

```diff
@@ -162,6 +167,19 @@ define(
                 }
             }
 
+            sortByTimeSystem(timeSystem) {
+                this.sortBy({
+                    key: timeSystem.key,
+                    direction: 'asc'
+                });
+                let formatter = this.openmct.telemetry.getValueFormatter({
+                    key: timeSystem.key,
+                    source: timeSystem.key,
+                    format: timeSystem.timeFormat
+                });
+                this.parseTime = formatter.parse.bind(formatter);
```

Comment:
The SortedTableRowCollection is supposed to be a generic sortable collection. It shouldn't have any knowledge of time built into it. That's the job of the `TelemetryCollection` (which is replacing the role of the `BoundedTableRowCollection`)

Reply:
Yep, removed!